src/types/params/public/ecdh.rs

```diff
@@ -24,6 +37,10 @@ pub enum EcdhPublicParams {
         p: x25519_dalek::PublicKey,
         hash: HashAlgorithm,
         alg_sym: SymmetricKeyAlgorithm,
+
+        /// Optional additional field for "forwardee keys", see
+        /// <https://www.ietf.org/archive/id/draft-wussler-openpgp-forwarding-00.html#name-generating-the-forwardee-ke>
+        replacement_fingerprint: Option<[u8; 20]>,
```

Comment:
given this is public (and a breaking change 😭 ) we should probably feature gate it